yellowbrick/text/dispersion.py

```diff
@@ -41,9 +47,16 @@ class DispersionPlot(TextVisualizer):
     ax : matplotlib axes, default: None
         The axes to plot the figure on.
 
-    color : list or tuple of colors
-        Specify color for bars
+    labels : list of strings
+        The names of the classes in the target, used to create a legend.
+        Labels must match names of classes in sorted order.
+
+    colors : list or tuple of colors
+        Specify the colors for each individual class
 
+    colormap : string or matplotlib cmap
+        Sequential colormap for continuous target
```

Comment:
Continuous targets are associated with regression problems, but I think the goal for this visualizer is to be used on classification problems, right? If the user is going to pass in a colormap here, we may want to suggest a qualitative colormap rather than a sequential one, since that will probably be easier to discern classes.

Reply:
@rebeccabilbro Sorry, The previous wording got carried over from the Tsne visualizer, I will change the wording to reflect your suggestion of using a qualitative colormap...

Reply:
Awesome, thank you!